fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1421,6 +1459,20 @@ ACTOR Future<Void> assignMutationsToStorageServers(CommitBatchContext* self) {
 					MutationRef backupMutation(
 					    MutationRef::Type::ClearRange, intersectionRange.begin, intersectionRange.end);
 
+					// TODO (Nim): Encrypt properly by splitting clear range across tenant boundaries
```

Comment:
nit:  please remove and elaborate on `Encrypt properly` , to a reader it gives a feeling as `encryption` policy is half-done or not followed at all.